svg/global_attributes.json

```
@@ -1438,6 +1441,41 @@
           }
         }
       },
+      "font-width": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": "18.4"
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror",
+            "webview_ios": "mirror"
+          },
+          "status": {
+            "experimental": false,
```

Shouldn't the linter complain when setting `experimental` to `false` despite only a single browser implementing the feature?

```suggestion
            "experimental": true,
```

---

I think non-standard takes precedence (which this probably is, see https://github.com/w3c/svgwg/issues/978). My hunch here is that `font-width` was added in Safari accidentally, so it's a mistake not an experiment.

---

There are [several instances](https://github.com/search?q=repo%3Amdn%2Fbrowser-compat-data+%2F%22experimental%22%3A+true%2C%5Cs*%22standard_track%22%3A+false%2C%5Cs*%22deprecated%22%3A+false%2F&type=code) of `experimental: true` alone.

So I think we should also set `experimental: true` here.